website/content/docs/connect/config-entries/service-resolver.mdx

```diff
@@ -526,6 +586,58 @@ spec:
           type: 'array<string>',
           description: 'A fixed list of datacenters to try during failover.',
         },
+        {
+          name: 'Targets',
+          type: 'array<ServiceResolverFailoverTarget>',
+          description: `A fixed list of failover targets to try during
+                          failover. It allows operators to express complicated failover
+                          scenarios such as between admin partitions, datacenters and cluster
+                          peers.`,
+          children: [
+            {
+              name: 'Service',
+              type: 'string: ""',
+              description:
+                'The service to use instead of the default of the failover target. If empty the current service is used.',
+            },
+            {
+              name: 'ServiceSubset',
+              type: 'string: ""',
+              description:
+                'The named subset to use for the failover target. If empty the default subset for the requested service is used.',
+            },
+            {
+              name: 'Namespace',
+              enterprise: true,
+              type: 'string: ""',
+              description:
+                'The namespace to use for the failover target. If empty the default namespace is used.',
+            },
+            {
+              name: 'Partition',
+              enterprise: true,
+              type: 'string: ""',
+              description:
+                'The admin partition to use for the failover target. If empty the default partition is used.',
+            },
+            {
+              name: 'Datacenter',
+              type: 'string: ""',
+              description:
+                'The datacenter to use for the failover target. If empty the current datacenter is used.',
+            },
+            {
+              name: 'Peer',
+              type: 'string: ""',
+              description:
+              `Specifies the destination cluster peer to resolve the service
+                  from. When peer is specified, the failover target relies on Envoy's
+                  outlier detection for health checks. Consul relies on outlier
+                  detection because imported service health checks don't account
+                  for service routers, splitters and resolvers.`,
```

Comment:
```suggestion
              `Specifies the destination cluster peer to resolve the service
                  from. When the peer is specified, the failover target relies on Envoy's
                  outlier detection for health checks. Consul relies on outlier
                  detection because imported service health checks do not account
                  for service routers, splitters and resolvers.`,
```